aggregator/pom.xml

```diff
@@ -131,7 +131,7 @@
         <dependency>
             <groupId>org.jenkins-ci.plugins</groupId>
             <artifactId>mailer</artifactId>
-            <version>1.12</version>
+            <version>1.13-SNAPSHOT</version>
```

Comment:
Depends on https://github.com/jenkinsci/mailer-plugin/pull/16. You could try

    mvn -DskipTests clean deploy

from a checkout of that branch, then set this version to the timestamped snapshot, something like `1.13-20150116.193108-1` (you will see the output from `deploy:deploy`). This allows us to pin this PR to a particular revision of the upstream PR, and should allow the PR builder to actually run on this one.